hack/cri-test/test-utils.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+. ${ROOT}/versions
+
+# POUCH_FLAGS are the extra flags to use when start pouchd.
+POUCH_FLAGS=${POUCH_FLAGS:-""}
+# RESTART_WAIT_PERIOD is the period to wait before restarting pouchd/containerd.
+RESTART_WAIT_PERIOD=${RESTART_WAIT_PERIOD:-10}
+
+POUCH_SOCK=/var/run/pouchcri.sock
+
+pouch_pid=
+containerd_pid=
+
+# test_setup starts containerd and pouchd.
+test_setup() {
+  local report_dir=$1
+
+  # Start containerd
+  if [ ! -x "$(command -v containerd)" ]; then
+    echo "containerd is not installed, please run hack/make.sh"
+    exit 1
+  fi
+  sudo pkill containerd
+  keepalive "containerd" ${RESTART_WAIT_PERIOD} &> ${report_dir}/containerd.log &
+  containerd_pid=$!
```

Comment:
line 39-41, please check the existence of containerd first, kill it only if containerd doesn't exist.